### PR TITLE
operators hive-operator (1.2.3179-0756b70e6)

### PR DESCRIPTION
**Pull request by abhinavdahiya**
Update Hive community operator to 1.2.3179-0756b70e6

/hold

### CODE REVIEW

File: operators/hive-operator/hive.package.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 channels:
-- currentCSV: hive-operator.v1.1.16
+- currentCSV: hive-operator.v1.2.3179-0756b70e6
```
Comment (joelddiaz) [inline operators/hive-operator/hive.package.yaml:2]: So we're going with (re)starting the alpha channel at 1.2.COMMIT_NUM-HASH to match what we're doing for ocm channel?

File: operators/hive-operator/1.2.3179-0756b70e6/hive-operator.v1.2.3179-0756b70e6.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,428 @@
...
+    capabilities: Seamless Upgrades
+    categories: OpenShift Optional
+    certified: 'false'
+    containerImage: quay.io/openshift-hive/hive:v1.2.3179-0756b70e6
+    createdAt: '2021-09-13T14:02:43Z'
+    description: OpenShift cluster provisioning and management at scale.
+    operators.operatorframework.io/internal-objects: '["checkpoints.hive.openshift.io","clusterdeprovisions.hive.openshift.io","clusterprovisions.hive.openshift.io","clusterstates.hive.openshift.io","machinepoolnameleases.hive.openshift.io","clustersyncleases.hiveinternal.openshift.io","clustersyncs.hiveinternal.openshift.io","fakeclusterinstalls.hiveinternal.openshift.io"]'
+    repository: https://github.com/openshift/hive
+    support: Hive Team
+  name: hive-operator.v1.2.3179-0756b70e6
```
Comment (2uasimojo) [inline operators/hive-operator/1.2.3179-0756b70e6/hive-operator.v1.2.3179-0756b70e6.clusterserviceversion.yaml:15]: ✓

File: operators/hive-operator/1.2.3179-0756b70e6/hive-operator.v1.2.3179-0756b70e6.clusterserviceversion.yaml [hunk 3]
```diff
@@ -0,0 +1,428 @@
...
+    url: https://www.openshift.com/blog/openshift-hive-cluster-as-a-service
+  - name: OpenShift
+    url: https://www.openshift.com/
+  maintainers:
+  - email: openshift-hive-team@redhat.com
+    name: Hive Team
+  maturity: alpha
+  provider:
+    name: Red Hat
+  replaces: hive-operator.v1.1.16
```
Comment (2uasimojo) [inline operators/hive-operator/1.2.3179-0756b70e6/hive-operator.v1.2.3179-0756b70e6.clusterserviceversion.yaml:427]: ✓

File: operators/hive-operator/hive.package.yaml (hunk 1 above)
Comment (2uasimojo) [reply operators/hive-operator/hive.package.yaml:2]: ✓

File: operators/hive-operator/hive.package.yaml [hunk 4]
```diff
@@ -11,7 +11,7 @@ channels:
   name: ocm-2.2
 - currentCSV: hive-operator.v1.1.9-1.1.8.2
   name: ocm-2.3
-- currentCSV: hive-operator.v1.1.16
+- currentCSV: hive-operator.v1.2.3179-0756b70e6
```
Comment (2uasimojo) [inline operators/hive-operator/hive.package.yaml:14]: ✓

**Review by 2uasimojo — COMMENTED** [review]
lgtm

File: operators/hive-operator/hive.package.yaml (hunk 1 above)
Comment (2uasimojo) [reply operators/hive-operator/hive.package.yaml:2]: > So we're going with (re)starting the alpha channel at 1.2.COMMIT_NUM-HASH to match what we're doing for ocm channel?

The prerelease OCM channel (currently 2.4) will get the same bundles as `alpha` -- though the OCM channel is allowed to lag behind -- until we branch the next release. At that point we'll start naming the release bundles according to the OCM release, like 2.4.COMMIT_NUM-HASH, and the _new_ prerelease OCM channel will carry on following `alpha` with 1.2.x.

**Pull request merged**